src/applications/personalization/profile-2/msw-mocks.js

```diff
@@ -97,9 +97,9 @@ const createTransactionRequestSuccessBody = {
 // couple of places but for testing purposes I'm reusing this body for _all_
 // transaction creation requests.
 const createTransactionRequestFailedError = {
-  title: 'Check Email Domain',
-  code: 'VET360_EMAIL304',
-  detail: 'AlphaNumeric ToplevelDomainName must be <= 63 Characters.',
+  title: 'Confirmation Date can not be greater than sourceDate',
```

Comment:
I needed to change this to a code other than EMAIL_304, otherwise the unit tests dependent on this value would fails since the AlertBox copy would return `It looks like the email you entered isn’t valid. Please enter your email address again.` 😄